src/main/resources/templates/index.twig

```diff
@@ -120,7 +120,7 @@
                     </div>
                     <div id="index-button-container">
                         <button id="student-login-button" type="button" class="btn btn-primary">I'm a Student</button>
-                        <button id="instructor-login-button" type="button" class="btn btn-primary">I'm a Teacher
+                        <button id="instructor-login-button" type="button" class="btn btn-primary">I'm an Instructor
```

Comment:
This change was actually requested in #64, but we've added it here for demo purposes.